src/pages/home/report/ReportActionItemContentCreated.tsx

```diff
@@ -51,7 +52,7 @@ function ReportActionItemContentCreated({contextValue, parentReportAction, trans
 
     const {report, action, transactionThreadReport} = contextValue;
 
-    const [policy] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY}${report.policyID ?? '-1'}`);
+    const [policy] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY}${report.policyID === CONST.POLICY.OWNER_EMAIL_FAKE ? -1 : report.policyID ?? '-1'}`);
```

Comment:
Why do we use `-1` (an int) when the policyID is fake, but `'-1'` (a string) if policyID is undefined? Should we use an int both times or a string both times to be consistent?

Reply:
Both int and string are fine here since we're using `` but it makes sense to make it consistent.

Reply:
@Beamanator I updated.